.github/workflows/scheduled-tasks.yml

```
@@ -0,0 +1,284 @@
+      - name: Configure kubectl
+        run: |
+          # Setup kubectl config
+          echo "${{ secrets.KUBE_CONFIG }}" | base64 -d > /tmp/kubeconfig
+          export KUBECONFIG=/tmp/kubeconfig
+
+      - name: Collect metrics
+        run: |
+          # Business metrics
+          kubectl exec -n zippicks-prod deployment/zippicks-api -- \
+            wp zippicks metrics:aggregate --type=business --period=hourly
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)